### DISCUSSION TITLE
Question: Is that anyway that i can signed pdf after set password?

### DISCUSSION THREAD

**Discussion by chayakornjj** [py-pdf/pypdf#708] [Q&A]
[Context]
- I want to use digital signature and pdf protected for invoice, Now that i can sign the digital signature or set the password of pdf only, I can't do both.

After research seem like it recommend me to set password first. after that before encrypt it need to sign it first..

[Question]

- Any one can give me an example or any idea?

Thanks

**Comment by MartinThoma** [comment]
Did you find an answer?
